puppet/modules/slave/manifests/packaging/rpm.pp

```diff
@@ -8,12 +8,13 @@
 ) {
   # TODO: Fix on EL8 and get rid of this
   $is_el8 = $facts['os']['release']['major'] == '8'
+  $is_el9 = $facts['os']['release']['major'] == '9'
 
   package { ['koji', 'rpm-build', 'createrepo', 'copr-cli']:
     ensure => installed,
   }
 
-  unless $is_el8 {
+  unless $is_el8 or $is_el9 {
```

Comment:
No annex, no pkg builds. While not needed for your task today, we should try using https://git-annex.branchable.com/install/rpm_standalone/ on el8/9.

Reply:
There doesn't seem to be any other options I can find other than us packaging it up into our infra repository. Or using Fedora nodes :smile:

Reply:
Updated this to use the repo -- take a look